### PR TITLE
tp-qemu: migration_after_nichotplug: some updates about session.

### PR DESCRIPTION
**Pull request by CongLi**
1. re-get session after re-enabling the primary link.
2. fix a typo of session.

Signed-off-by: Cong Li <coli@redhat.com>

id: 1128017

### CODE REVIEW

**Comment by xutian** [comment]
double check this case, there still have session issue at line#74

**Comment by CongSmile** [comment]
15:56:31 INFO | PASS 1-smp_4.4096m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.nic_hotplug.migration.nic_8139

@xutian Could you help review it again?

Thanks.

File: qemu/tests/migration_after_nichotplug.py [hunk 1]
```diff
@@ -52,9 +52,9 @@ def set_link(nic_name, up=False):
     # Modprobe the module if specified in config file
     module = params.get("modprobe_module")
     if guest_is_not_windows and module:
-        session.cmd_output("modprobe %s" % module)
-    if session:
-        session.close()
+        session_serial.cmd_output("modprobe %s" % module)
```
Comment (xutian) [inline qemu/tests/migration_after_nichotplug.py]: session_serial.cmd() is enough.

Comment (CongLi) [inline qemu/tests/migration_after_nichotplug.py]: Updated.

**Comment by PandaWei** [comment]
ACK.

**Pull request merged**